test/orm/declarative/test_tm_future_annotations_sync.py

```diff
@@ -2260,6 +2260,29 @@ class Foo(decl_base):
         )
         is_(Foo.__table__.c.status.type.native_enum, expected_native_enum)
 
+    @testing.requires.python312
+    def test_pep695_literal_defaults_to_enum(self, decl_base):
```

Comment:
I am not sure if this is the right place, but this is a modified example of the the following test (from the same module). https://github.com/sqlalchemy/sqlalchemy/blob/68ba6eceab1de5daa6108b9ebbe6f4f4fe1e596b/test/orm/declarative/test_tm_future_annotations_sync.py#L2228-L2261

I did slight modifications to make it 3.12 only and used the new type alias syntax.

Reply:
you likely want to add this test to `test_typed_mapping` instead, since this file is automatically generated

https://github.com/sqlalchemy/sqlalchemy/blob/82803016b5fcbc3225af87a43768dbea2be87582/test/orm/declarative/test_tm_future_annotations_sync.py#L1-L6

Reply:
thank you, I have done this :)